### ISSUE TITLE
1.30.0 error

### ISSUE THREAD

**Issue by dingshaohua-com** [jooy2/vitepress-sidebar#187]
The error is as follows
使用pnpm，报错如下

```js
> book-doc-vitepress@1.0.0 docs:dev
> vitepress dev docs

failed to load config from /Users/dsh/files.localized/code/book-doc/src/apps/vitepress/docs/.vitepress/config.mts
failed to start server. error:
Cannot find module '/Users/dsh/files.localized/code/book-doc/node_modules/.pnpm/vitepress-sidebar@1.30.0/node_modules/vitepress-sidebar/dist/util' imported from /Users/dsh/files.localized/code/book-doc/node_modules/.pnpm/vitepress-sidebar@1.30.0/node_modules/vitepress-sidebar/dist/sidebar.js
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/dsh/files.localized/code/book-doc/node_modules/.pnpm/vitepress-sidebar@1.30.0/node_modules/vitepress-sidebar/dist/util' imported from /Users/dsh/files.localized/code/book-doc/node_modules/.pnpm/vitepress-sidebar@1.30.0/node_modules/vitepress-sidebar/dist/sidebar.js
    at finalizeResolution (node:internal/modules/esm/resolve:257:11)
    at moduleResolve (node:internal/modules/esm/resolve:913:10)
    at defaultResolve (node:internal/modules/esm/resolve:1037:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:650:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:599:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:582:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:241:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:132:49)
```

**Comment by dingshaohua-com** [comment]
😂 测试出来了，跟pnpm没关系。 
作者你给改坏了啊，上一个版本 1.29.0 还是好的，你对他做了什么啊。

**Comment by dingshaohua-com** [comment]
你是不是提取类型声明文件的时候，向外暴漏的错了

**Comment by jooy2** [comment]
Hello, I apologize for the inconvenience caused by the recent changes. It was my mistake for not checking it in depth after deployment.

This issue was fixed in version `1.30.2`, and upgrading the package to that version or higher should fix the issue.

Thank you for reporting the issue.

**Comment by dingshaohua-com** [comment]
> 您好，对于最近的更改给您带来的不便，我深表歉意。这是我的错误，在部署后没有深入检查。
> 
> 该问题已在版本中修复`1.30.2`，将软件包升级到该版本或更高版本应该可以解决该问题。
> 
> 感谢您报告此问题。

Haha, I actually wrote a [sidebar generation plugin for vuepress](https://github.com/dingshaohua-com/vuepress-theme-sidebar) a long time ago. But it is no longer maintained. Anyway, applaud you for making vitepress more useful.

**Comment by dingshaohua-com** [comment]
> Hello, I apologize for the inconvenience caused by the recent changes. It was my mistake for not checking it in depth after deployment.
> 
> This issue was fixed in version `1.30.2`, and upgrading the package to that version or higher should fix the issue.
> 
> Thank you for reporting the issue.

Haha, I actually wrote a [sidebar generation plugin for vuepress](https://github.com/dingshaohua-com/vuepress-theme-sidebar) a long time ago. But it is no longer maintained. Anyway, applaud you for making vitepress more useful.
